pkg/skaffold/deploy/kubectl/kubectl.go

```diff
@@ -87,20 +87,21 @@ func NewDeployer(cfg Config, labels map[string]string, provider deploy.Component
 	}
 
 	podSelector := kubernetes.NewImageList()
+	kubectl := NewCLI(cfg, d.Flags, defaultNamespace)
```

Comment:
Should this code not be using `cfg.GetKubeNamespace()` somehow?

Reply:
to be honest I'm not sure. I used `cfg.GetKubeNamespace()` in the other deployers because there wasn't a `defaultNamespace` to use, but in the `Runner` originally the empty string was being passed. maybe I should change it for consistency?

Reply:
ah, looking at the code again, `kubectl.NewCLI` actually uses `cfg.GetKubeNamespace()` as the fallback if t adefault namespace isn't provided. so we should be good here

Reply:
ah ... I suppose that's ok, though `""` is supposed to be equivalent to `default`.

Reply:
right, the `defaultNamespace` is actually something that can be specified by the user (different from the "default" of `""`)